managed-databases/postgresql-and-mysql/how-to/upgrade-version.mdx

```diff
@@ -14,36 +14,42 @@ categories:
+    - **Upgrade and switch incoming traffic**: With this method your endpoint is migrated to the new Database Instance automatically. This option will create a clone of your Database Instance, and will automatically migrate the endpoint to the new Instance. The original Database Instance remains available, and will continue to be billed. The endpoint will be deleted from the original Database Instance.
 
-    <Lightbox src="scaleway-rdb-engine-version-upgrade.webp" alt="List of Database Instances in the Scaleway console. A warning sign displays under the name of the Database Instance that has an End of Life engine version. Next to the warning, an upgrade hyperlink displays" />
-2. Click **Upgrade** next to the Database Instance you wish to upgrade the database engine for. A pop-up appears.
-    <Message type="tip">
-      Alternatively you can click the database name or <Icon name="more" /> > **More info**, then click **Upgrade** under **Database engine** in the Database Instances information page.
-    </Message>
 
-    <Message type="note">
-      The **Upgrade version** option is only available for:
-        - Database Instances using End of Life engine versions.
-        - Database Instances using Block volumes. If your Database Instance uses a local volume, you must first [change the volume type](/managed-databases/postgresql-and-mysql/how-to/change-volume-type) to a Block volume and try again.
+    <Message type="important">
+      When you upgrade to a new version, there is no synchronisation between the source and target Database Instances. To avoid data loss, we recommend you stop any write operations running on your applications during the upgrade. If you let them run during the process, the data will be stored only in the source Database Instance.
     </Message>
 
-    <Lightbox src="scaleway-engine-version-upgrade-pop-up.webp" alt="" />
-3. Make sure that you have read and agree with the conditions of the upgrade.
-
-4. Click **Upgrade version**. A new Database Instance is created.
-    <Message type="important">
-      When you upgrade a version, a clone of your current Database Instance with an up-to-date engine version is created. Your current Database Instance will remain available with its original endpoints, and will still be billed.<br /><br />The new, upgraded Database Instance will have new endpoints. To use it, replace the old endpoints with the new across all your applications. The upgrade does not impact your original Database Instance. Even after manually updating your endpoints, you can roll back to the old ones at any time.
+    <Message type="tip">
+      Follow the [migrating endpoints via the CLI](/managed-databases/postgresql-and-mysql/api-cli/migrating-endpoints) procedure to quickly migrate your endpoints. You can use this procedure to revert the migration anytime.
```

Comment:
We can add a note there. Reverting the endpoint won't have effect on the date stored on the database. Meaning of some entries were added in the upgraded database, they won't be added back on the old version if the client choose to migrate back its endpoints